example/urls.py

```diff
@@ -1,33 +1,34 @@
 from example import views
 
 from django.conf import settings
-from django.conf.urls import static, url
+from django.conf.urls import static
 from django.contrib import admin
+from django.urls import re_path
 
 urlpatterns = [
-    url(r"^$", views.thumbnail_options, name="thumbnail_options"),
-    url(
+    re_path(r"^$", views.thumbnail_options, name="thumbnail_options"),
```

Comment:
As Django 2.2 is still supported we are not quiet ready to drop support for it. So it would be good to make this change in backwards comaptible way so that the tests for 2.2 don't stumble accros `re_path`. [This](https://github.com/jonasundderwolf/django-image-cropping/commit/e8d63c230#diff-dbc982bab16c85f6038e0031d4b635dc9fd023b279c99e8dc57bf3e0b925d739L51) is how we approached similar issues in the past.

Reply:
re_path is available in Django 2.2:
https://docs.djangoproject.com/en/2.2/ref/urls/#django.urls.re_path

Reply:
Oh, i missed that in the docs. This makes things even simpler, great!